azure-pipelines.yml

```diff
@@ -63,7 +63,11 @@ jobs:
       Windowslatest:
         vmImage: 'windows-2019'
         cli11.std: 20
-        cli11.options: -DCMAKE_CXX_FLAG="/std:c++latest"
+        cli11.options: -DCMAKE_CXX_FLAGS="/std:c++latest /EHsc"
```

Comment:
@phlptp This was not working before, due to a typo (missing S). Now we need EHsc, maybe because this replaces the default flags? Just checking to make sure it is fine.

Reply:
That explains a few things.  I suppose in general requiring /EHsc could be potentially problematic but probably not for anything CLI11 does by itself.